dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/RuntimeUpdateListenerImpl.java

```diff
@@ -103,40 +119,54 @@ public void notificationCreated(RuntimeUpdateManager updateManager, RuntimeUpdat
 
                         Dictionary<String, Object> props = configuration.getProperties();
                         String[] libraryRefs = (String[]) props.get("libraryRef");
+                        String[] jCacheManagerRefs = (String[]) props.get("jCacheManagerRef");
 
-                        if (libraryRefs == null) {               
-                            if (!isSessionCacheBellConfigured()) {
-                                Tr.error(tc, "ERROR_CONFIG_INVALID_HTTPSESSIONCACHE", Tr.formatMessage(tc, "SESSION_CACHE_CONFIG_MESSAGE", sampleConfig));
-                            }
-                        } else if (libraryRefs.length == 0) {
-                            Tr.debug(tc, "The libraryRef attribute of the httpSessionCache in the server configuration could not be resolved. "
-                                            + "Check for possible CWWKG0033W messages.");
-                        } else {
-                            //verify that the libraryRef points to a valid library configuration object.
-                            Configuration libraryConfiguration = configAdmin.getConfiguration(libraryRefs[0]);
-                            if(libraryConfiguration != null) {
-                                //We found nothing wrong with the configuration.
-                                if (trace && tc.isDebugEnabled())
-                                    Tr.debug(this, tc, "A httpSessionCache configuration with a valid libraryRef was found.");
+                        if (jCacheManagerRefs == null) {
+                            if (libraryRefs == null) {
+                                if (!isSessionCacheBellConfigured()) {
+                                    Tr.error(tc, "ERROR_CONFIG_INVALID_HTTPSESSIONCACHE", Tr.formatMessage(tc, "SESSION_CACHE_CONFIG_MESSAGE", sampleConfig));
+                                }
+                            } else if (libraryRefs.length == 0) {
+                                Tr.debug(tc, "The libraryRef attribute of the httpSessionCache in the server configuration could not be resolved. "
+                                             + "Check for possible CWWKG0033W messages.");
```

Comment:
Should these be warning messages? Or throwing an error? Or does CWWKG0033W make better sense for the customer (not really sure of the context of what happens here)

Reply:
I kept these the same as the other messages that were already there.